app/Schedule/View.elm

```diff
@@ -269,6 +274,16 @@ predictionColor minutesLate =
             Color.red
 
 
+arrival : Date -> Train -> Node Msg
+arrival date train =
+    trainMetadata ("arrives at" ++ arrivalTime train)
+
+
+arrivalTime : Train -> String
+arrivalTime train =
+    prettyTime train.scheduledArrival
```

Comment:
@gfontenot mentioned this is officially called [Eta reduction](https://wiki.haskell.org/Eta_conversion)